src/types.ts

```diff
@@ -1,3 +1,11 @@
 export type Request = chrome.devtools.network.Request | chrome.devtools.network.HAREntry
 
-export const isChromeRequest = (request: Request) : request is chrome.devtools.network.Request => (typeof (request as chrome.devtools.network.Request).getContent === 'function')
\ No newline at end of file
+export const isChromeRequest = (request: Request) : request is chrome.devtools.network.Request => (typeof (request as chrome.devtools.network.Request).getContent === 'function')
+
+import type { Fact } from '@ucanto/interface'
+
+declare module '@ucanto/interface' {
+  interface Delegation<C> {
+    facts: Fact[]
```

Comment:
this shouldn't be needed - it already exists on the interface: https://github.com/storacha/ucanto/blob/d7420d5f96962a05652dd165c1fd26fa1d140f47/packages/interface/src/lib.ts#L243